apps/src/lib/levelbuilder/CourseOfferingEditor.jsx

```diff
@@ -46,6 +49,9 @@ export default function CourseOfferingEditor(props) {
   const [isSaving, setIsSaving] = useState(false);
   const [lastSaved, setLastSaved] = useState(null);
   const [thumbnail, setThumbnail] = useState(null);
+  const [selectedDate, setSelectedDate] = useState(
+    courseOffering.published_date ? moment(courseOffering.published_date) : null
```

Comment:
Just for my own knowledge, is `moment()` a date/time formatter or what is its purpose here?

Reply:
Yeah, because the datetime gets converted to a string when retrieved from the model, I used moment to turn the string into a moment object which the DatePicker can then use. I was getting an error when trying to use the string to create a Date object